src/app/util/attribute-storage.cpp

```diff
@@ -1411,3 +1443,22 @@ bool emberAfExtractCommandIds(bool outgoing, EmberAfClusterCommand * cmd, Cluste
     return true;
 }
 #endif
+
+void registerAttributeAccessInterceptor(app::AttributeAccessInterceptor * interceptor)
+{
+    interceptor->SetNext(attributeAccessInterceptors);
+    attributeAccessInterceptors = interceptor;
```

Comment:
Should we somehow generate an error if an interceptor targeting the same path already exists ? It should not happens in practice as there is a single reference cluster implementation but IIRC the firstly registered interceptor will be returned, it may result into unexpected result for the user.

Reply:
> Should we somehow generate an error if an interceptor targeting the same path already exists ? 

That's a good idea.  I will add that.

Reply:
Seems like a great thing to assert on.